pipelines/vars/forklift_plugins.yml

```
@@ -17,6 +17,7 @@ server_box:
         - "--enable-foreman-plugin-bootdisk"
         - "--enable-foreman-plugin-discovery"
         - "--enable-foreman-plugin-hooks"
+        - "{{ '--enable-foreman-plugin-puppet' if pipeline_version == 'nightly' or pipeline_version is version('3.0', '>=') else '' }}"
```

```suggestion
        - "{{ '--enable-foreman-plugin-puppet' if pipeline_version == 'nightly' or pipeline_version is version('3.1', '>=') else '' }}"
```

Or does it really matter? The plugin is already installed on 3.0, and doesn't need manual installation.

---

It was for not installing it prior 3.0 release... now it does not matter :)

---

or maybe not, lets be safe :)